libraries/block_production/pow_producer.cpp

```diff
@@ -9,6 +9,8 @@
 #include <koinos/crypto/multihash.hpp>
 #include <koinos/pack/classes.hpp>
 
+#define TARGET_BLOCK_INTERVAL_MS 10000
```

Comment:
This should be using `std::chrono` similar to `hashrate::update_interval`.  We also don't make assumptions about the PoW contract (like the target being a 10s block interval) and I'm not sure we should break that...